src/Jackett.Updater/Program.cs

```diff
@@ -193,6 +193,7 @@ private void ProcessUpdate(UpdaterConsoleOptions options)
                 "Definitions/dragonworld.yml",
                 "Definitions/hdclub.yml",
                 "Definitions/polishtracker.yml",
+                "Indexers/SceneAccess.cs",
```

Comment:
Oh, is this necessary for *.cs indexers, or does it only apply to *.yml definitions?